app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/autofill/AutoFillScreen.kt

```diff
@@ -146,6 +155,22 @@ fun AutoFillScreen(
                     .testTag("InlineAutofillSwitch")
                     .padding(horizontal = 16.dp),
             )
+            if (state.showPasskeyManagementRow) {
+                BitwardenExternalLinkRow(
+                    text = stringResource(id = R.string.passkey_management),
```

Comment:
@SaintPatrck Do you know if we need this to be feature flagged until all of passkeys is ready?

I assume we do not want this visible for the next beta release?

Reply:
Good catch! Yes, it should be hidden behind a feature flag. Passkey support is only enabled on debug builds at the moment.

Reply:
We're now approved to merge this!